lib/modules/manager/composer/utils.ts

```diff
@@ -46,6 +47,22 @@ export function getComposerArguments(
   return args;
 }
 
+export function getComposerUpdateArguments(
+  config: UpdateArtifactsConfig,
+  toolConstraint: ToolConstraint,
+): string {
+  let args = getComposerArguments(config, toolConstraint);
+
+  if (
+    is.string(toolConstraint.constraint) &&
+    api.intersects!(toolConstraint.constraint, '^2.7')
+  ) {
+      args += ' --minimal-changes';
+  }
```

Comment:
If `api.intersects()` can ever throw, it would be best to wrap this block in a try/catch and emit a `logger.warn()` if it ever throws (and then continue with default args)

Reply:
This function now never throws an exception, the `try`/`catch` block is inside it, so there is no need to do anything extra.